var/spack/repos/builtin/packages/openjpeg/package.py

```diff
@@ -43,5 +43,3 @@ class Openjpeg(CMakePackage):
     version('2.0',   'cdf266530fee8af87454f15feb619609')
     version('1.5.2', '545f98923430369a6b046ef3632ef95c')
     version('1.5.1', 'd774e4b5a0db5f0f171c4fc0aabfa14e')
-
```

Comment:
If we just take a step back and think about it, it's insane how many packages we can install with almost no code. There are probably hundreds of would be Autotools and CMake packages that no longer need any special install method. #1186 might just go down as the biggest line saving PR in Spack history. I'm looking at this empty package and it's hard to think of anything else we could remove. Now if we could just get rid of that stupid license and `from spack import *`...